### PR TITLE
SQL-865: Setting default property values for logging parameters

### PR DESCRIPTION
**Pull request by bucaojit**
Setting default values for `LogDir` and `LogLevel` properties in case it is not set in existing and created *.tds and *.twb files.

### CODE REVIEW

File: connector/connectionProperties.js [hunk 1]
```diff
@@ -5,13 +5,15 @@
     props["database"] = attr[connectionHelper.attributeDatabase];
     props["dialect"] = "mongosql";
 
-    if (attr["v-log-directory"] == null) {
-        props["LogDir"] = "console";
-    } else {
+    props["LogDir"] = "console";
+    if (attr["v-log-directory"] != null && !attr["v-log-directory"] === "") {
```
Comment (nbagnard) [inline connector/connectionProperties.js]: I tried out the new taco file and this does not work. I always end up with OFF and Console even when changing the LogLevel to FINER for example.
This works : `if (attr["v-log-directory"] != null && attr["v-log-directory"] !== "")`

File: connector/connectionProperties.js [hunk 2]
```diff
@@ -5,13 +5,15 @@
...
         props["LogDir"] = attr["v-log-directory"];
     }
 
-    props["LogLevel"] = attr["v-loglevel"];
+    props["LogLevel"] = "OFF";
+    if(attr["v-loglevel"] != null && !attr["v-loglevel"] === "") {
```
Comment (nbagnard) [inline connector/connectionProperties.js]: And similarly, here, you want to use : `if(attr["v-loglevel"] != null && attr["v-loglevel"] !== "")`

**Review by nbagnard — CHANGES_REQUESTED** [review]

File: connector/connectionProperties.js (hunk 1 above)
Comment (bucaojit) [reply connector/connectionProperties.js]: Thanks for checking.  I made the suggested change.

File: connector/connectionProperties.js (hunk 2 above)
Comment (bucaojit) [reply connector/connectionProperties.js]: Made this change as well.

**Review by nbagnard — APPROVED** [review]
LGTM!
No idea why your original code did not work ¯\_(ツ)_/¯
Maybe extra parenthesis around the negated clause would have worked too.

**Comment by bucaojit** [comment]
evergreen merge

**Pull request merged**